packages/SwingSet/src/kernel/vatManager/supervisor-subprocess-xsnap.js

```diff
@@ -194,23 +193,26 @@ function makeWorker(port) {
       testLog: (...args) => port.send(['testLog', ...args]),
     };
 
+    const cacheSize =
+      typeof virtualObjectCacheSize === 'number'
+        ? virtualObjectCacheSize
+        : undefined;
+
     const ls = makeLiveSlots(
       syscall,
       vatID,
       vatPowers,
       vatParameters,
-      // @ts-ignore  TODO: defend against non-numeric?
-      virtualObjectCacheSize,
+      cacheSize,
       // TODO: lsgc? API drift?
     );
 
     const endowments = {
       ...ls.vatGlobals,
       console: makeConsole(`SwingSet:vatWorker`),
       assert,
-      // @ts-ignore bootstrap provides HandledPromise
```

Comment:
I don't understand why this is no longer necessary. Help?

Reply:
If you use an explicit `globalThis.foo`, typescript doesn't complain, but it does if you just use `foo`.